server/app.py

```diff
@@ -284,11 +284,13 @@ def get_status(job_id):
         "RETRY": Status.INPROGRESS,
     }
     job = streaming_parent_task.AsyncResult(job_id)
-    logger.info(f"\n\nParent task meta : {job.info}\n\n")
+    logger.info(f"Parent task meta : {job.info}")
 
     # If parent task meta does not contain input/output job IDs - return FAILED
     if (
         job.info is None
+        or type(job.info)
+        != dict  # In some failure cases e.g. an exception is thrown, job.info will be a string, causing the next line to crash
```

Comment:
`black` linter made this a new line, looks weird but guess it works

Reply:
make sure you're running `poetry run black` instead of just `black` so you don't accidentally use a local version with different linting rules. Happened to me

Reply:
Ah I def did that lol good call

Reply:
No changes with running that way fyi